File: Assignment__19/Assignment_1.c
#include<stdio.h>
#include<stdlib.h>

int CountEven(int Arr[] , int iLength)
{
    int iCnt = 0 , iCount = 0 ;
    for(iCnt = 0 ; iCnt < iLength;iCnt++)
    {
        if(Arr[iCnt] % 2 == 0)
        {
            iCount = iCount + 1;
        }
    }
    return iCount;

}
int main()
{
    int iSize = 0 , iCnt = 0 ;
    int iRet = 0 ;
    int *ptr = NULL;

    printf("Enter number of Elements :  ");
    scanf("%d",&iSize);

    ptr = (int *)malloc(iSize * sizeof(int));
    if(ptr == NULL)
    {
        printf("Unable to locate memory : \n");
        return -1;
    }

    printf("Enter %d numbers : \n",iSize);
    for(iCnt = 0  ; iCnt < iSize ; iCnt++)
    {
        printf("Enter %d numbers : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }

    iRet = CountEven(ptr , iSize); 
    printf("Result is : %d \n",iRet);

    free(ptr);

    return 0 ;
}

File: Assignment__19/Assignment_java/Assignment_1.java

import java.util.*;

class CountEven
{
    public int Frequency(int [] Arr , int iLength)
    {
        int iCount = 0;
        for(int iCnt = 0 ; iCnt < iLength ; iCnt++)
        {
            if(Arr[iCnt] % 2  == 0 )
            {
                iCount = iCount + 1 ;
            }
           
        }
        return iCount;
    }
}
class Assignment_1 
{

    public static void main(String A [])
    {
        Scanner sobj = new Scanner(System.in);
       

        System.out.println("Enter number  : ");
        int iSize = sobj.nextInt();
        int []ptr = new int[iSize];

        System.out.println("Enter Elements  : "+iSize);

        for(int iCnt = 0 ; iCnt < iSize; iCnt++)
        {
            System.out.print("Enter Elements : "+(iCnt + 1)+" : ");
            ptr[iCnt] = sobj.nextInt();

        }
        CountEven dobj = new CountEven();
        int iRet = dobj.Frequency(ptr ,iSize);
        System.out.println("Result is : "+iRet);
    }
}